src/components/App/QRGenerator/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useQRCode } from 'react-qrcodes';
+
+const QRCode = ({ text }: { text: string }) => {
+  const options = {
+    level: 'M',
+    margin: 4,
+    scale: 4,
+    width: 4,
+    color: {
+      dark: '#000000ff',
+      light: '#ffffffff'
+    }
+  };
+
+  const [qrRef] = useQRCode({
+    text: text,
+    options: options
+  });
+
+  return <canvas ref={qrRef} />;
+};
+
+const QRGenerator = () => {
+  const [text, setText] = React.useState('');
+  return (
+    <>
+      <input
+        style={{ width: 100, height: 20 }}
```

Comment:
We should avoid inline styling imo, make classes when you need styles - in the interest of getting this merged, the styling doesn't need to look perfect in order to be merged, but it should be a separate `.scss` file for sure